src/site/antora/modules/ROOT/pages/manual/configuration.adoc

```diff
@@ -2219,18 +2216,6 @@ The following is a list of available global configuration properties. Note that
   until a listener is registered. In practice, a listener is registered when a configuration is found,
   and from that point onwards, status messages are only sent to the listeners (depending on their statusLevel).
 
-| [[defaultStatusLevel]]log4j2.defaultStatusLevel
-  ([[Log4jDefaultStatusLevel]]Log4jDefaultStatusLevel)
-| LOG4J_DEFAULT_STATUS_LEVEL
-| ERROR
-|
-  The StatusLogger logs events that occur in the logging system to the console.
-  During configuration, AbstractConfiguration registers a StatusConsoleListener with the StatusLogger that may
-  redirect status log events from the default console output to a file.
-  The listener also supports fine-grained filtering.
-  This system property specifies the default status log level for the listener to use if the configuration does not specify a status level.
-  Note: this property is used by the log4j-core implementation only after a configuration file has been found.
-
 | [[statusLoggerLevel]]log4j2.statusLoggerLevel
   ([[log4j2.StatusLogger.level]]log4j2.StatusLogger.level)
 | LOG4J_STATUS_LOGGER_LEVEL
```

Comment:
This is incorrect, the default is `ERROR, right?

Reply:
Nice catch!

Somehow users never found this error in all these years, maybe because `log4j2.statusLoggerLevel` was always broken.

Reply:
Done